_includes/partners/x-col.html

```diff
@@ -1,13 +1,12 @@
 <article class="container {{ include.class }}">
   <header class="intro"><h2>{{ include.heading }}</h2></header>
   <div class='grid-container'>
-    <div class="grid-row grid-gap-lg">
+    <div class="grid-row grid-gap-lg" aria-label="{{ include.label }}">
```

Comment:
This was an addition that I'm wondering if this would be helpful?

Reply:
My understanding is that `aria-label` on static content like this can have some inconsistent behavior across different screen readers, and may also override any content within. Might be okay if we'd want to optimize for the grouping label, but fall back to allow unsupporting screen readers to list out the individual, labelled images?

http://www.davidmacd.com/blog/does-aria-override-static-backup.html

Or may not be worth the trouble, since labelled images on their own could be fine as well.